database/src/main/scala/cromwell/database/sql/CallCachingStore.scala

```diff
@@ -7,7 +7,7 @@ import scala.concurrent.{ExecutionContext, Future}
 trait CallCachingStore {
   def addToCache(metaInfo: CallCachingResultMetaInfoEntry, hashes: Iterable[HashKeyAndValue], result: Iterable[ResultSimpleton])(implicit ec: ExecutionContext): Future[Unit]
   def metaInfoIdsMatchingHashes(hashKeyValuePairs: Set[HashKeyAndValue])(implicit ec: ExecutionContext): Future[Set[MetaInfoId]]
-  def fetchCachedResult(metaInfoId: MetaInfoId)(implicit ec: ExecutionContext): Future[Option[CachedResult]]
+  def fetchCachedResult(metaInfoId: MetaInfoId)(implicit ec: ExecutionContext): Future[CachedResult]
```

Comment:
I can revert this to return an `Option[CachedResult]`, but what should this do for a `None`?